### PR TITLE
fix: update app init

### PR DESCRIPTION
**Pull request by acezard**
Post breaking changes from dependencies

Cozy-flags introduced breaking changes which were not detected earlier. This has been fixed in this mr. Also the dependency tree has been cleaned a bit.

All dependencies in the project are now up to date to their latest minor versions (except two).

No dependency has been updated to its next major version though as it seems risky to do everything at once.

### CODE REVIEW

File: src/lib/debug.jsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { useEffect } from 'react'
-import { useFlag } from 'cozy-flags'
+import useFlag from 'cozy-flags/dist/useFlag'
```
Comment (acezard) [inline src/lib/debug.jsx:2]: Not a good look but could not get it imported any other way

File: src/targets/browser/index.ejs [hunk 2]
```diff
@@ -1,32 +1,45 @@
...
+      content="width=device-width, height=device-height, initial-scale=1, viewport-fit=cover"
+    />
+    <% _.forEach(htmlWebpackPlugin.files.css, function(file) { %>
+    <link rel="stylesheet" href="<%- file %>" />
+    <% }); %> <% _.forEach(htmlWebpackPlugin.files.js, function(file) { %>
+    <script src="<%- file %>" defer></script>
+    <% }); %> <% if (__TARGET__ === 'mobile') { %>
+    <meta name="format-detection" content="telephone=no" />
+    <script src="cordova.js" defer></script>
+    <% } else { %>
```
Comment (acezard) [inline src/targets/browser/index.ejs:40]: this is the only non-prettier change here

File: src/lib/debug.jsx [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+import FlagSwitcher from 'cozy-flags/dist/FlagSwitcher'
```
Comment (acezard) [inline src/lib/debug.jsx:3]: Same as above

File: src/targets/browser/index.ejs (hunk 2 above)
Comment (Crash--) [reply src/targets/browser/index.ejs:40]: Why did you remove the __STACK_ASSETS__ test? 

Since cozy-bar is still in 7.X, I don't think there is a BC somewhere

Comment (acezard) [reply src/targets/browser/index.ejs:40]: Somehow it prevented the bar from loading, not sure why?

Comment (Crash--) [reply src/targets/browser/index.ejs:40]: This is not the right fix then. 

What is the error when you revert it?

Comment (acezard) [reply src/targets/browser/index.ejs:40]: window.cozy.bar === undefined

Comment (acezard) [reply src/targets/browser/index.ejs:40]: maybe it was a side-effect of cozy-flags, let me try again

Comment (acezard) [reply src/targets/browser/index.ejs:40]: image

nope still broken even after having fixed cozy-flags

Comment (acezard) [reply src/targets/browser/index.ejs:40]: linked to this? 
image

File: src/lib/debug.jsx (hunk 1 above)
Comment (JF-Cozy) [reply src/lib/debug.jsx:2]: Est-ce qu'on pourrait pas regarder du côté de cozy-flag si on peut corriger le souci ?

Comment (acezard) [reply src/lib/debug.jsx:2]: Possible oui mais cette MR est assez urgente car master est broken. @Crash-- que préfères-tu ?

Comment (Crash--) [reply src/lib/debug.jsx:2]: Créons l'issue sur le répo de cozy-flag et on traitera ça plus tard

Comment (Crash--) [reply src/lib/debug.jsx:2]: C'est vraiment important cette notion de si on a un problème suite à une maj de lib de faire l'issue sur la lib pour : 
- tracker le souci 
- pouvoir le corriger
- partager le fait qu'il y a un souci 

Garder ça dans son coin c'est pas productif pour cozy, car on tombera toujours sur le même souci et on reperdra encore et encore du temps sur le sujet

**Review by Crash-- — APPROVED** [review]

File: src/lib/debug.jsx (hunk 1 above)
Comment (acezard) [reply src/lib/debug.jsx:2]: issue created https://github.com/cozy/cozy-libs/issues/1330
